packages/lab/src/calendar/internal/CalendarDay.css

```diff
@@ -1,59 +1,53 @@
-.uitk-light .uitkCalendarDay {
-  --calendar-day-current-day-border-color: var(--uitk-color-grey-900);
-}
-
-.uitk-dark .uitkCalendarDay {
-  --calendar-day-current-day-border-color: var(--uitk-color-white);
-}
-
 /* TODO: Design need to align characteristics for CalendarDay */
 .uitkCalendarDay {
-  --calendar-day-text-color: var(--uitk-text-primary-foreground);
-  --calendar-day-background: var(--uitk-selectable-background);
-  --calendar-day-background-hover: var(--uitk-selectable-background-hover);
-  --calendar-day-text-color-hover: var(--uitk-selectable-foreground-hover);
-  --calendar-day-out-of-range-text-color: var(--uitk-text-secondary-foreground);
-  --calendar-day-blocked-text-color: var(--uitk-selectable-foreground);
-  --calendar-day-blocked-cursor: var(--uitk-selectable-cursor-disabled);
-  --calendar-day-blocked-icon-color: var(--uitk-status-error-foreground);
-  --calendar-day-blocked-background: var(--calendar-day-background);
-  --calendar-day-unselectable-text-color: var(--uitk-text-secondary-foreground-disabled);
-  --calendar-day-unselectable-background: var(--calendar-day-background);
-  --calendar-day-unselectable-cursor: var(--uitk-selectable-cursor-disabled);
-
-  --calendar-day-selected-background: var(--uitk-selectable-background-selected);
-  --calendar-day-selected-text-color: var(--uitk-selectable-foreground-selected);
-  --calendar-day-selected-focused-outline-color: var(--uitk-color-white);
-
-  --calendar-day-selected-start-background: var(--uitk-selectable-background-selected);
-  --calendar-day-selected-start-text-color: var(--uitk-selectable-foreground-selected);
-  --calendar-day-selected-start-focused-outline-color: var(--uitk-color-white);
-
-  --calendar-day-selected-end-background: var(--uitk-selectable-background-selected);
-  --calendar-day-selected-end-text-color: var(--uitk-selectable-foreground-selected);
-  --calendar-day-selected-end-focused-outline-color: var(--uitk-color-white);
-
-  --calendar-day-selected-span-background: var(--uitk-selectable-background-hover);
-  --calendar-day-selected-span-text-color: var(--uitk-selectable-foreground);
-
-  --calendar-day-hovered-span-background: var(--uitk-selectable-background-hover);
-  --calendar-day-hovered-span-text-color: var(--uitk-selectable-foreground);
-
-  --calendar-day-hovered-offset-background: var(--uitk-selectable-background-hover);
-  --calendar-day-hovered-offset-text-color: var(--uitk-selectable-foreground);
+  --calendarDay-text-color: var(--uitk-text-primary-foreground);
```

Comment:
we should keep this as --calendar-day as Calendar is the component and day is the subject - i've made this standard across components with a lint rule too in https://github.com/jpmorganchase/uitk/pull/335

Reply:
Technically the component is CalendarDay? Or are we naming this inline with the public component